ang/workflow/action-links/directives/workflow-list-edit-action.html

```diff
@@ -1 +1 @@
-<a>{{ ts('Edit') }}</a>
+<a ng-controller="WorkflowEditController" ng-click="clickHandler(workflow.id)">{{ ts('Edit') }}</a>
```

Comment:
Is there a reason to create a full controller and tests for this simple redirection that can be done using `href` ?

Reply:
In RSE-1280, we need to redirect to another page based on some conditions. Hence I started creating the Controller which later be useful to add more logic to it.